common/src/test/java/tachyon/util/io/BufferUtilsTest.java

```diff
@@ -365,4 +365,21 @@ public void cleanDirectBufferTest() {
       Assert.fail("cleanDirectBuffer is causing memory leak." + ooe.getMessage());
     }
   }
+
+  @Test
+  public void sliceByteBufferTest() {
+    final int size = 100;
+    final ByteBuffer buf = ByteBuffer.allocate(size);
```

Comment:
it would be nice to test with data, e.g. use `BufferUtils#getIncreasingByteBuffer` and make sure the data in the slice is correct.